Cargo.lock

```diff
@@ -680,9 +680,9 @@ dependencies = [
 
 [[package]]
 name = "compiler_builtins"
-version = "0.1.55"
+version = "0.1.65"
```

Comment:
In case anyone's wondering: yes, I made a typo when publishing a new version of compiler-builtins.